interfaces/apparmor/template.go

```diff
@@ -220,6 +218,7 @@ var defaultTemplate = []byte(`
   /etc/machine-id r,
   /etc/mime.types r,
   @{PROC}/ r,
+  @{PROC}/version_signature r,
```

Comment:
It's weird that @{PROC}/version and @{PROC}/version_signature aren't next to each other like they were before but I won't block on that. Feel free to update it without needing me to look at this again.